specification/riskiq/Easm/tspconfig.yaml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  "service-dir":
+    default: "sdk/easm"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-directory: "examples"
+    output-file: "{azure-resource-provider-folder}/Microsoft.Easm/preview/2023-03-01-preview/easm.json"
```

Comment:
You should use the variables for version and version type, like this:

```yaml
output-file: "{azure-resource-provider-folder}/Microsoft.Easm/{version-status}/{version}/easm.json"
```

as shown here: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml#L18